lib/src/iterable_matchers.dart

```diff
@@ -152,9 +152,11 @@ Matcher unorderedMatches(Iterable expected) => new _UnorderedMatches(expected);
 
 class _UnorderedMatches extends Matcher {
   final List<Matcher> _expected;
+  final bool _mustMatchEveryValue;
```

Comment:
nit: I don't like this name, it sounds like not all of `expected` have to match or something which wouldn't make sense.... maybe just name it more directly what it does, `_checkLength`.

Reply:
sure